examples/dev_tools/fps_overlay.rs

```diff
@@ -0,0 +1,63 @@
+//! Showcase how to use and configure FPS overlay.
+
+use bevy::{
+    dev_tools::fps_overlay::{FpsOverlayConfig, FpsOverlayPlugin},
+    prelude::*,
+};
+
+fn main() {
+    App::new()
+        .add_plugins((
+            DefaultPlugins,
+            FpsOverlayPlugin {
+                config: FpsOverlayConfig {
+                    // Here we define size of our overlay
+                    font_size: 50.0,
+                    // We can also change color of the overlay
+                    font_color: Color::srgb(0.0, 1.0, 0.0),
+                    // If we want, we can use a custom font
+                    font_path: None,
+                },
+            },
+        ))
+        .add_systems(Startup, setup)
+        .add_systems(Update, system)
+        .run();
+}
+
+fn setup(mut commands: Commands) {
+    // We need to spawn camera to see overlay
+    commands.spawn(Camera2dBundle::default());
+    commands.spawn(
+        TextBundle::from_sections([
+            TextSection::new(
+                "Press 1 to change color of the overlay.",
+                TextStyle {
+                    font_size: 25.0,
+                    ..default()
+                },
+            ),
+            TextSection::new(
+                "\nPress 2 to change size of the overlay",
+                TextStyle {
+                    font_size: 25.0,
+                    ..default()
+                },
+            ),
+        ])
+        .with_style(Style {
+            justify_self: JustifySelf::Center,
+            ..default()
+        }),
+    );
+}
+
+fn system(input: Res<ButtonInput<KeyCode>>, mut overlay: ResMut<FpsOverlayConfig>) {
```

Comment:
I think we should rename that function just so we make sure the user can understand everything properly.
`toggle_config` is a possibility